drivers/adc/Kconfig.rtl87x2g

```diff
@@ -0,0 +1,16 @@
+# RTL87X2G uart configuration
+
+# Copyright(c) 2020, Realtek Semiconductor Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config ADC_RTL87X2G
+	bool "RTL87X2G MCU adc driver"
+	default y
+	depends on DT_HAS_REALTEK_RTL87X2G_ADC_ENABLED
+	select REALTEK_ADC_ADC
+    select REALTEK_USING_SDK_LIB
+	help
+	  This option enables the adc driver for RTL87X2G family of
+	  processors.
+	  Say y if you wish to use adc on RTL87X2G MCU.
```

Comment:
"RTL87X2G family of processors" seems inappropriate, " the rtl87x2g series SoC" may be better?
Do you mean "Set y", rather than "Say y"?

Reply:
modified